set.mm

```diff
@@ -68486,6 +68486,13 @@ result of an operator (deduction version).  (Contributed by Paul
       BDMZUCUAEKZTUCUEUBEKUAUBEEDHNOUETAEKCAEFGIPOQTUDUCJRS $.
   $}
 
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Convert operation laws using setvar variables to class notation
```

Comment:
```suggestion
  Variable-to-class conversion for operations
```
Our section titles are nouns (or nominal phrases), and not verbal phrases.

I propose "variable-to-class" since "vtocl*" are common labels.

Reply:
Done.